src/TensorFlowNET.Core/Binding.cs

```diff
@@ -7,5 +7,10 @@ namespace Tensorflow
     public static partial class Binding
     {
         public static tensorflow tf { get; } = New<tensorflow>();
+
+        /// <summary>
+        ///     Alias to null, similar to python's None.
+        /// </summary>
+        public static readonly object None = null;
```

Comment:
Why not define it as `int None = -1` ?

Reply:
I've decided to make Binding.None to be null - only seems logical to follow python's logic in this case.

None is used in many many cases in TF and only when it is in a shape is when we need it as -1. in all other cases it is supposed to be null.